x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -144,6 +144,7 @@
     public static final String JOB_AUDIT_SNAPSHOT_STORED = "Job model snapshot with id [{0}] stored";
     public static final String JOB_AUDIT_REVERTED = "Job model snapshot reverted to ''{0}''";
     public static final String JOB_AUDIT_SNAPSHOT_DELETED = "Model snapshot [{0}] with description ''{1}'' deleted";
+    public static final String JOB_AUDIT_SNAPSHOTS_DELETED = "Model snapshots {0} with descriptions {1} deleted";
```

Comment:
Maybe we could just say `{0} expired model snapshots deleted` (where `{0}` is how many).  We could be deleting 24 per day if they've been changed to be every hour, and that would make the message very verbose.  The fact that these are the ones that have expired should be OK for the audit log.

Reply:
@droberts195 diggin' it